Amplitude/Amplitude.m

```diff
@@ -991,32 +1082,37 @@ - (void)startSession
     return;
 }
 
-#pragma mark - configurations
-
-- (void)setUserProperties:(NSDictionary*) userProperties
+- (void)identify:(AMPIdentify *)identify
 {
-    [self setUserProperties:userProperties replace:NO];
+    if (identify == nil || [identify.userPropertyOperations count] == 0) {
+        return;
+    }
+    [self logEvent:IDENTIFY_EVENT withEventProperties:nil withApiProperties:nil withUserProperties:identify.userPropertyOperations withTimestamp:nil outOfSession:NO];
 }
 
-- (void)setUserProperties:(NSDictionary*) userProperties replace:(BOOL) replace
+#pragma mark - configurations
+
+- (void)setUserProperties:(NSDictionary*) userProperties
```

Comment:
I didn't remove the one with the replace. I just moved it to line 1113, and made it call this one.